nexus/src/nexus.rs

```diff
@@ -3666,6 +3667,57 @@ impl Nexus {
             .await?;
         Ok(db_silo_user)
     }
+
+    // SSH public keys
+
+    /// Following GitHub, we do not bother to authorize this request.
```

Comment:
While true, I think it makes sense to have this true by policy rather than implementation?  That is, still do an authz check saying that we're listing the ssh keys of user "x", and just have the Oso policy say "all authenticated users can do that" if we want that?  (But also: I think we said for now we'd say only a user can see their own ssh keys.  Either way seems fine to me, though I'd like another +1 from somebody else if we do make these public.)

Reply:
Fixed in 95114e05.